tests/test_core_multimap.py

```diff
@@ -1,6 +1,7 @@
 from typing import Sequence, NamedTuple
 
-from amazon.ion.core import Multimap
+# from amazon.ion.core import Multimap
+from amazon.ion.ion_py_objects import IonPyDict_new as Multimap  # replace the original multiMap with the new IonPyDict
```

Comment:
Okay. I'm also working on adding more tests.

Reply:
I created a separate test file - `test_multimap_IonPyDict` with the same unit tests. The only difference is that the new tests use `IonPyDict` instead of previous `multimap`.